Jenkinsfile

```diff
@@ -140,24 +133,14 @@ for (int i = 0; i < splits.size(); i++) {
                     // but not letting the build to fail will cause next build not to try those tests again.
                     allowEmptyResults: true
                     ) {
-                      sh """
-                          set-java.sh ${jdk}
-                          # Ensure that Jenkins node setup does not influence the container java setup
-                          unset JAVA_HOME
-                          eval \$(vnc.sh)
-                          java -version
-                          mvn -v
-                          run.sh ${browser} ${jenkinsVersion} -Dmaven.repo.local=${WORKSPACE_TMP}/m2repo -Dmaven.test.failure.ignore=true -Dcsp.rule -DforkCount=1 -B
-                          cp --verbose target/surefire-reports/TEST-*.xml /reports
-                          """
+                      sh "./ci.sh ${jdk} ${browser} ${jenkinsVersion}"
```

Comment:
It would be easier to read the build logs if the building of the container was separated out from the running of it.

There's 2300 lines from building of the image.
I only care about that when it fails generally and if it was in another step it would be cleaner imo

Reply:
Sure, but that problem also exists before this PR.

Reply:
Not from what I can tell, the image was either built separately or reusing and existing built image.

This PR changes it to use `ath-container.sh` that wasn't used before in CI.

Reply:
I don't understand what "built separately or reused and existing built image" means.

When the `build-image` label was used before this PR, an image was built, also resulting in many lines of log output.

Reply:
Before this change:
https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-2019/1/pipeline-overview/?selected-node=108

<img width="858" alt="image" src="https://github.com/user-attachments/assets/b4a6c2b6-39e9-4f1e-9808-8b211f74dc10" />

<img width="841" alt="image" src="https://github.com/user-attachments/assets/99f8ac3a-751f-44fd-83b6-640396f37ab6" />

After this change:
<img width="851" alt="image" src="https://github.com/user-attachments/assets/8c911f60-ccc2-4c31-9ff4-3c91c31c4487" />

-----

Basically there's a lot more log output to go through, that is introduced in this PR

Reply:
> Basically there's a lot more log output to go through in a single step rather than before it was in a separate docker build step, that is introduced in this PR

Please read my previous comment. That has been addressed in the latest revision.

Reply:
Looks good thanks